src/pal/inc/pal_corefx.h

```diff
@@ -45,6 +45,31 @@ ForkAndExecProcess(
            int* stdoutFd,
            int* stderrFd);
 
+struct corefx_stat {
+    int64_t dev;     /* ID of device containing file */
+    int32_t mode;    /* protection */
+    int32_t uid;     /* user ID of owner */
+    int32_t gid;     /* group ID of owner */
+    int64_t size;    /* total size, in bytes */
+    int64_t atime;   /* time of last access */
+    int64_t mtime;   /* time of last modification */
+    int64_t ctime;   /* time of last status change */
+};
```

Comment:
Some systems include a birthtime, i.e. time of creation, and we would like to expose that if it's available.  Should we include that?